runners/apex/pom.xml

```diff
@@ -229,10 +229,51 @@
               </ignoredUsedUndeclaredDependencies>
             </configuration>
           </execution>
+          <execution>
+            <id>dependency-tree</id>
```

Comment:
Comment here about where it is used, so we can keep track and make sure they stay in sync.